Accelerometer/CS/README.md

```diff
@@ -0,0 +1,298 @@
+# SPI Accelerometer
+
+We'll connect an SPI accelerometer to your UpBoard and create a simple app to read data from it. We'll walk you through step-by-step, so no background knowledge of SPI is needed.
+However, if you're curious, SparkFun provides a great [tutorial on SPI](https://learn.sparkfun.com/tutorials/serial-peripheral-interface-spi).
+
+### Load the project in Visual Studio
+
+You can find the source code for this sample by downloading a zip of all of our samples. Make a copy of the folder on your disk and open the project from Visual Studio 2019.  
+
+### Connect the SPI Accelerometer to your device
+
+You'll need a few components:
+
+* <a name="SPI_Accelerometer"></a>an [ADXL345 accelerometer board from Sparkfun](https://www.sparkfun.com/products/9836) with pin headers soldered on
+
+* a breadboard and a couple of male-to-female connector wires
+
+Visit the **UpBoard** sections below depending on which device you have:
+
+![Electrical Components](../../Resources/components.png)
+
+#### UpBoard
+If you have a UPBOARD, we need to hook up power, ground, and the SPI lines to the accelerometer.

```

Comment:
UPBOARD is all capitals here, UpBoard or Up-Board elsewhere.  Please be consistent.  Also it shoudl be "an UpBoard" not "a UpBoard"

Reply:
Made the changes as mentioned in the readme document.